src/Microsoft.CSharp/src/MultilingualResources/Microsoft.CSharp.de.xlf

```diff
@@ -0,0 +1,520 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is a substantial amount of machine-generated XML!  Could we instead adopt an approach where if the file isn't available, it's generated as a temporary during build and then thrown away?  That way we'd only need to check in the files for which manual editing was necessary.  We could potentially even make it so that you only had to check in override content, with everything else filled in by the build system automatically.  I'm nervous about the fact that, for a single project, this change is adding 8000 lines of code.  We have ~100 projects in corefx.  Back of the napkin estimate, and this means ~800,000 lines of boilerplate eventually checked in to the repo.